test/fn/move-contacts.spec.js

```diff
@@ -214,6 +214,57 @@ describe('move-contacts', () => {
     });
   });
 
+  it('move district_1 to flexible hierarchy root type', async () => {
```

Comment:
Can we please have a test where we move a flexible hierarchy contact?

Reply:
yep!